### ISSUE TITLE
Plugin link in "writing servers" is an unwanted distraction

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3772]
|<img alt="mattj's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| mattj reported|
|-|-|
|Trac ID|trac#3772|
|Type|enhancement|
|Created|2009-04-12 05:39:50Z|
|Branch|https://github.com/twisted/twisted/tree/writing-server-3772-2|

The Writing Servers page is a fairly good intro, though "you should read this document second, after first reading the top level overview of how to begin writing your Twisted application, in Writing Plug-Ins for Twisted" doesn't seem to be very good advice.

Someone reading the Writing Servers doc probably just wants to figure how a basic server works, so it would be better if the docs about plugins etc (which don't have much preface either) were left until later.

<details><summary>Searchable metadata</summary>

```
trac-id__3772 3772
type__enhancement enhancement
reporter__mattj mattj
priority__normal normal
milestone__ 
branch__branches_writing_server_3772_2 branches/writing-server-3772-2
branch_author__thijs thijs
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__documentation documentation
time__1239514790000000 1239514790000000
changetime__1309195292000000 1309195292000000
version__None None
owner__ 
cc__exarkun cc__cary cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @glyph|
|-|-|

The plugin document used to explain things other than how to write plugins.  It got updated to document the new plugin system, though, and all of the stuff about how to structure a Twisted application was removed.  I think it would be appropriate for the server howto to link to either application.xhtml or tap.xhtml or both now.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [27111]) Branching to 'writing-servers-3772'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [27995]) Branching to 'writing-server-3772-2'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

(In [27997]) Add link to tap.xhtml, refs [#3772](https://github.com/twisted/twisted/issues/3772)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_8"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Are there really people who fit the tap.xhtml target audience of "those that have developed a Twisted application which needs a command line-based deployment mechanism." who haven't already read the server howto? I think the server howto should be the first thing you read no matter what if you're new to Twisted, in which case the line in servers.xhtml containing the link in question should just go away, in my mind.

Other than that, there are 2 spots of hidden whitespace in server.xhtml that could go away.

Thanks for all the activity on documentation tickets!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Thanks jesstess.

**Comment by twisted-trac** [comment]
|<img alt="binjured's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_10"></a>|binjured commented|
|-|-|

Any movement or next steps to be completed here?

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_12"></a>|@thijstriemstra set status to `closed`|
|-|-|

Replying to [binjured](#note_10):
> Any movement or next steps to be completed here?

This was fixed with  [#4761](https://github.com/twisted/twisted/issues/4761) so I'm marking this as a duplicate.
